dnf/cli/output.py

```diff
@@ -20,6 +20,7 @@
 from __future__ import absolute_import
 from __future__ import print_function
 from __future__ import unicode_literals
+from dnf.cli.format import format_date, format_datetime
```

Comment:
Just for the `format_date` and `format_datetime` calls it won't fulfil the condition of at least five references.